tests/providers/gcp/gcp_test.py

```diff
@@ -0,0 +1,1070 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file no longer exists since the tests were refactored in e0ccf13185d1a5002e82a8b23a2d4ba09307aaf2